### PR TITLE
chore(macro): Refactor currency converters

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

File: server/macros/currency_usd_to_cad.py [hunk 1]
```diff
@@ -37,21 +37,9 @@ def convert(match):
...
-
-    if item.get('abstract', None):
-        item['abstract'] = re.sub('\$([0-9]+)', convert, item['abstract'])
-
-    if item.get('headline', None):
-        item['headline'] = re.sub('\$([0-9]+)', convert, item['headline'])
-
-    if item.get('slugline', None):
-        item['slugline'] = re.sub('\$([0-9]+)', convert, item['slugline'])
+    for field in ['body_html', 'body_text', 'abstract', 'headline', 'slugline']:
```
Comment (plamut) [inline server/macros/currency_usd_to_cad.py]: Using `set()` is better than the list.

**Comment by plamut** [comment]
:+1:

**Pull request merged**